src/crab/var_id_map.hpp

```
@@ -82,6 +82,24 @@ class VarIdMap {
         return id_to_var_.size();
     }
 
+    /// Rename variables in-place. The underlying IDs and their relationships
+    /// are unchanged — only the Variable labels are swapped.
+    void rename(const std::vector<std::pair<Variable, Variable>>& renaming) {
```

Clang (14.0.6)

[warning] all parameters should be named in a function

(readability-named-parameter)